code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -206,6 +206,9 @@
 		else
 			to_chat(user, "You detach \the [helmet] from \the [src]'s helmet mount.")
 			helmet.loc = get_turf(src)
+			if(istype(src,/obj/item/clothing/head/helmet/space/hardsuit/syndi))
+				var/obj/item/clothing/head/helmet/space/hardsuit/syndi/S = helmet
+				S.linkedsuit = null
 			src.helmet = null

```

Comment:
Missed this. Remove src., it is implicit

Reply:
mfw merged anway